### PR TITLE
Website UI updated

### PR DESCRIPTION
**Pull request by ayushmanyd**
feat added: #49

### Website UI updated -
- Homepage added.
- Route changed for scanMutualFunds.
- About Us added.

#### Screenshot:

- New Homepage:
image

- Route changed for scanMutualFunds:
image

@shivamsouravjha @ananupam

### CODE REVIEW

**Comment by shivamsouravjha** [comment]
[Animation - 1729078921033.json](https://github.com/user-attachments/files/17395272/Animation.-.1729078921033.json)
@ayushmanyd the animation

**Comment by ayushmanyd** [comment]
Animation looks good. Is there anything you want me to add or change? Otherwise, you can merge if you want. @shivamsouravjha

**Comment by shivamsouravjha** [comment]
Also add an google analytics event catcher

**Comment by ayushmanyd** [comment]
I think this will work, you just have to setup your Google Tag Manager.

You can test this on https://tagassistant.google.com
It's working or not, you have to check because it can only be checked from website domain name not from localhost (I tried but didn't worked).

File: src/pages/homepage.jsx [hunk 1]
```diff
@@ -13,24 +13,42 @@ const Homepage = () => {
...
-  }, []);
+    return () => anim.destroy() // Cleanup on unmount
+  }, [])
+
+  const handleButtonClick = () => {
+    window.gtag('event', 'button_click', {
+      event_category: 'button',
+      event_label: 'Discover More',
+      value: 1,
+    })
```
Comment (shivamsouravjha) [inline src/pages/homepage.jsx:26]: currently we use     `ReactGA.send({ hitType: 'pageview', page: '/homepage' })`  I guess

**Comment by shivamsouravjha** [comment]
I've added it already just had to be used here.

**Comment by ayushmanyd** [comment]
Can you please add levels, It'll be a great help. @shivamsouravjha

**Pull request merged**